extensions/zerion/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
 # Zerion Changelog
 
+## [Zerion Client ] - 2024-12-17
```

Comment:
```suggestion
## [Zerion Client] - {PR_MERGE_DATE}
```

The date can be `{PR_MERGE_DATE}` here. The Raycast bot will replace the string with merge date for you.

And there is trailing space after the `Zerion Client`. It can be removed.